bin/push_docker.js

```diff
@@ -30,36 +28,41 @@ const currentBranch = execSync('git rev-parse --abbrev-ref HEAD')
   .toString()
   .trim();
 
-const distributionParam = process.argv[2];
-const stageParam = process.argv[3];
-const suffix = distributionParam ? `-${distributionParam}` : '';
-const stage = stageParam ? `-${stageParam}` : '';
+const distributionParam = process.argv[2] || '';
+const stageParam = process.argv[3] || '';
 const commitSha = process.env.GITHUB_SHA || 'COMMIT_ID';
 const commitShaLength = 7;
 const commitShortSha = commitSha.substring(0, commitShaLength - 1);
-const configurationEntry = `wire-web-config-default${
-  suffix ? suffix : currentBranch === 'main' ? '-main' : '-staging'
+const configurationEntry = `wire-web-config-default-${
+  distributionParam ? distributionParam : currentBranch === 'main' ? 'main' : 'staging'
 }`;
-const dependencies = {
-  ...appConfigPkg.dependencies,
-  ...appConfigPkg.devDependencies,
-  ...appConfigPkg.peerDependencies,
-  ...appConfigPkg.optionalDependencies,
-};
 
-const configVersion = dependencies[configurationEntry].split('#')[1];
```

Comment:
Does it make sense to (A) simplify it that way ([app-config as no other fields](https://github.com/wireapp/wire-account/blob/staging/app-config/package.json)), and (B) to port this to the other two repos?